src/build_cache.ts

```diff
@@ -111,7 +111,7 @@ export class ProdBuildCache implements BuildCache {
       : path.join(base, pathname);
 
     // Check if path resolves outside of intended directory.
-    if (path.relative(base, filePath).startsWith(".")) {
+    if (path.relative(base, filePath).startsWith("..")) {
```

Comment:
I can't come up with a scenario where just `.` and not `..` can escape to a parent directory except for e.g. `./../`. But if this is deemed "risky", we could perhaps change it to:

```ts
    const rel = path.relative(base, filePath);
    if (rel.startsWith(".") && !rel.startsWith(".well-known/") {
```

The `.well-known` folder is the only example I can come up with currently that is required to start with `.`, though there are likely cases somewhere 😄

Reply:
yeah it's fine. The `../foo` case is the one to guard against. I guess I was just lazy when writing that code and checked for a single dot.

Reply:
Good, then I wasn't misunderstanding something! 🙌